frontend/src/metabase/components/Schedule/Schedule.module.css

```diff
@@ -2,25 +2,33 @@
   line-height: 2.5rem;
   display: grid;
   grid-template-columns: fit-content(100%) auto;
-  gap: 0.5rem;
-  row-gap: 0.35rem;
+  gap: var(--schedule-grid-gap, 0.5rem);
+  row-gap: var(--schedule-row-gap, 0.35rem);
+  font-weight: var(--schedule-font-weight, normal);
```

Comment:
I think these vars are only used once, so maybe you could get away with using the values without introducing the vars

Reply:
@rafpaf If I understand correctly, you propose to just reassign these properties in `.schedule` class inside  `NotificationSchedule.module.css`? This could work, but I believe using this variable-based approach would be more future-proof if for some reason the structure of `Schedule` changes and its internal styling becomes more specific for some reason.

Reply:
I'll keep it, since you don't mind. The thing is – we're already using flexible styling, as you can see. But this approach should be a bit more safe if, say, tomorrow someone adjusts the styling of `Schedule` by introducing a more specific selector, and doesn't check how `NotificationsSchedule` looks.

Generally, I consider CSS variables as another dimension of Component's API. And I think Mantine has the same approach. If we need to adjust internal styling, we use the API. Otherwise, when we need to adjust positioning or margins, we use classnames.